### PR TITLE
feat: add Google calendar online events

### PR DESCRIPTION
**Pull request by LeonVreling**
- adds a new menu entry with the name "Online events", this redirects to "my.aegee.eu/online/events"
- imports the Google calendar on the new page

### CODE REVIEW

File: src/views/dashboard/OnlineEvents.vue [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<template>
+  <div>
+    <div class="content">
+
+      <iframe src="https://calendar.google.com/calendar/b/4/embed?height=800&amp;wkst=1&amp;bgcolor=%23ffffff&amp;ctz=Europe%2FBrussels&amp;src=YWVnZWUuZXVfdjVtbjY1MWltZXFwdnM4N3Y0bG43aHIxZjRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ&amp;color=%237CB342&amp;showCalendars=0&amp;showTabs=1&amp;showPrint=1&amp;showDate=1" style="border:none; height:700px" width="100%" frameborder="0" scrolling="no"></iframe>
```
Comment (serge1peshcoff) [inline src/views/dashboard/OnlineEvents.vue:5]: did you check out the ical thing that I've sent? is it working?

Comment (LeonVreling) [reply src/views/dashboard/OnlineEvents.vue:5]: Ah no I forgot about that one, can you send it again maybe cause I can't find it back now

Comment (serge1peshcoff) [reply src/views/dashboard/OnlineEvents.vue:5]: https://stackoverflow.com/questions/30766477/google-calendar-ical-url-for-public-calendar I took a reference from here and just replaced the calendar URL with the one from the iframe

File: package.json [hunk 2]
```diff
@@ -32,6 +32,8 @@
     "buefy": "^0.8.9",
     "chart.js": "^2.9.3",
     "dompurify": "^2.0.7",
+    "fetch": "^1.1.0",
```
Comment (serge1peshcoff) [inline package.json:35]: this one is not required

File: src/views/dashboard/Calendar.vue [hunk 3]
```diff
@@ -72,7 +72,6 @@ export default {
       this.fetchEvents()
     },
     eventClick (info) {
```
Comment (serge1peshcoff) [inline src/views/dashboard/Calendar.vue:74]: this one in general is not required

Comment (LeonVreling) [reply src/views/dashboard/Calendar.vue:74]: yes I was still working on showing the description of the events in some way, as they don't have a URL to redirect on click and give more information

Comment (serge1peshcoff) [reply src/views/dashboard/Calendar.vue:74]: if there's a meet link we can use it I suppose, what do you think?

Comment (LeonVreling) [reply src/views/dashboard/Calendar.vue:74]: yes or a link to the Facebook event, that is most often also in the description, but then you would have to write a function to parse the HTML data in the description and look for that specific word

File: src/views/dashboard/Calendar.vue [hunk 4]
```diff
@@ -2,12 +2,15 @@
   <div>
     <div class="content">
       <div class="tags">
-        <span class="tag" v-for="(type, key) in constants.EVENT_TYPES_NAMES" :key="key" :style="{ 'background-color': colors[key], 'color': textColors[key] }">
+        <span class="tag" v-for="(type, key) in constants.EVENT_TYPES_NAMES" :key="key" :style="{ 'background-color': colors[key], 'color': '#FFFFFF' }">
```
Comment (serge1peshcoff) [inline src/views/dashboard/Calendar.vue:5]: this one shouldn't be changed

Comment (serge1peshcoff) [reply src/views/dashboard/Calendar.vue:5]: actually why not, they are all white

**Comment by serge1peshcoff** [comment]
@LeonVreling is it final and working? if so I will merge

**Comment by LeonVreling** [comment]
Yes it is all done now 👌

**Comment by serge1peshcoff** [comment]
let's deploy then!

**Pull request merged**